### PR TITLE
Byzcoin protocol simulation doesn't run anymore

### PR DESCRIPTION
**Pull request by ineiti**
Implements #551

### CODE REVIEW

File: protocols/protocols.go [hunk 1]
```diff
@@ -38,7 +38,7 @@ import (
 	_ "github.com/dedis/cothority/protocols/randhound"
 	// ByzCoin has some strange library which uses 'seelog' that doesn't
 	// free all go-routines
-	//_ "github.com/dedis/cothority/protocols/byzcoin"
-	//_ "github.com/dedis/cothority/protocols/byzcoin/ntree"
-	//_ "github.com/dedis/cothority/protocols/byzcoin/pbft"
+	_ "github.com/dedis/cothority/protocols/byzcoin"
```
Comment (Liamsi) [inline protocols/protocols.go]: There was a reason we commented these 3 lines out (see comment on top). I think we should keep it this way and add a comment in the ByzCoin simulation pointing to these lines?

Comment (ineiti) [inline protocols/protocols.go]: True...

**Comment by Liamsi** [comment]
LGTM

**Comment by Liamsi** [comment]
But maybe add a pointer to these 3 lines? So @LefKok doesn't need to ask us how-to compile it in the future?

**Comment by ineiti** [comment]
I'll also add the downloading of the block-file and md5-verification.

**Pull request closed without merging**